jujugui/static/gui/src/app/components/deployment-flow/deployment-flow.js

```diff
@@ -266,13 +266,12 @@ YUI.add('deployment-flow', function() {
      @method _handleLogin
     */
     _handleLogin: function() {
-      const controllerAPI = this.props.controllerAPI;
-      controllerAPI.connect();
-      const handler = controllerAPI.after('facadesChange', e => {
-        if (e.newVal && controllerAPI.get('facades')) {
```

Comment:
So we had this b/c we were concerned about a race condition that @hatched had encountered--I'll see if I run into it in QA; are you reasonably certain no such concern exists?

I defer entirely to you and @hatched on this one--the vagaries of the controller are still not something I understand too well.